app/src/main/java/com/xmartlabs/template/model/UserRequest.kt

```diff
@@ -0,0 +1,7 @@
+package com.xmartlabs.template.model
+
+data class UserRequest(
+    var name: String = "",
```

Comment:
same for the others string we put null because in case that nothing is assign gson would not send this attribute